verification/curator-service/api/src/controllers/sources.ts

```diff
@@ -97,6 +97,12 @@ export default class SourcesController {
                 });
                 return;
             }
+            // Undefined fields are removed from the request body by openapi
+            // validator, if we want to unset the dateFilter we have to pass an
+            // empty object and set it undefined ourselves here.
```

Comment:
:/ should we do this for other fields too?

Reply:
This is the only "struct", other singular fields like strings are passed as empty strings and get through as expected.
The automation struct is handled separately.